test/handle_test.cpp

```diff
@@ -28,6 +28,10 @@
 /// \todo Create dedicated ticket and rename macro.
 #define WORKAROUND_SWDEV_257056_PCH_MISSING_MACROS 1
 
+// https://gerrit-git.amd.com/c/compute/ec/clr/+/972441
+#define WORKAROUND_ISSUE_2600 \
+    (HIP_PACKAGE_VERSION_FLAT > 5007023384ULL && HIP_PACKAGE_VERSION_FLAT <= 6000023494ULL)
```

Comment:
@junliume Are we sure that HIPRTC fix will arrive to HIP runtime right after 6.0.23494?

Reply:
Yes that's the info provided to me, the patch from hipRTC has merged but not yet in 6.0

Reply:
[Recommendation] Okay. Anyway, we'll see the failure and adjust the upper bound if necessary. But let's prepare for potential 5.7.2 (do not disable this test for it), what do you think?
```suggestion
    (HIP_PACKAGE_VERSION_FLAT > 6000000000ULL && HIP_PACKAGE_VERSION_FLAT <= 6000023494ULL)
```

Reply:
I'll do this in my `fix-hiprtc-60`.